govcd/adminorg_administration.go

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2020 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package govcd
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+	"github.com/vmware/go-vcloud-director/v2/util"
+)
+
+// LdapConfigure allows to configure LDAP mode in use by the Org
+func (adminOrg *AdminOrg) LdapConfigure(settings *types.OrgLdapSettingsType) error {
```

Comment:
I haven't found test for it. Also we usually we return entity which we created/updated

Reply:
added